index.html

```diff
@@ -288,5 +288,18 @@ <h2>Contact Me</h2>
         </div>
       </form>
     </section>
+
+    <section class="mobile-menu-wrapper mobile-menu-js hide">
```

Comment:
- [ ] @Team great work on the mobile menu, but your background doesn't match the design given, text in it is not blurred, below is a comparison, kindly fix it:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/194895527-59571c20-6194-41bb-849a-4b3dd2306255.png) |  ![image](https://user-images.githubusercontent.com/61408860/194895595-655f4dd3-de13-4424-8b91-aad28ce76b47.png)